core/templates/components/profile-link-directives/profile-link-image-backend-api.service.ts

```diff
@@ -33,8 +33,12 @@ export class ProfileLinkImageBackendApiService {
   constructor(
     private http: HttpClient
   ) {}
-
-  async fetchProfilePictureDataAsync(profileImageUrl: string): Promise<string> {
+  // It will return a 'null' Promise in the following cases:
+  // 1. The user is not logged in
+  // 2. The user didn't uploaded a profile picture
+  // 3. The player is in preview mode.
```

Comment:
Who is 'player' in the two previous points you're talking about 'user'.

Reply:
![Screenshot from 2021-06-14 10-14-43](https://user-images.githubusercontent.com/43031234/121842198-69453b80-ccf9-11eb-9871-0abf7a68dc5c.png)
It is referring to the above when the player in the exploration editor is in preview mode.

Reply:
Okay, try to explain it in the comment in a better way then.

Reply:
done